code/src/java/pcgen/io/PCGFile.java

```diff
@@ -167,6 +168,6 @@ public static boolean isPCGenCharacterOrPartyFile(final File file)
 	 */
 	private static String getWindowsSafeFilename(final File file)
 	{
-		return file.getName().toLowerCase();
+		return file.getName().toLowerCase(Locale.ENGLISH);
```

Comment:
Isn't this almost certainly dangerous?  File names are user controlled...

Reply:
fixed. I originally was thinking that system files should use `Locale.English` but re-reading it makes more sense to explicitly use the default locale.